Controls/SerialOutputCoT.cs

```diff
@@ -171,7 +171,7 @@ void mainloop()
 
                             if (CoTStream != null && CoTStream.IsOpen)
                             {
-                                CoTStream.WriteLine(xmlStr.Replace("\r", ""));

```

Comment:
I can't remember why I added this but it doesn't really do anything and doesn't belong in this PR.